hack/tools/go.mod

```diff
@@ -1,72 +1,79 @@
 module github.com/metal3-io/ip-address-manager/hack/tools
 
-go 1.22.3
+go 1.22.7
 
-require (
-	github.com/jteeuwen/go-bindata v3.0.7+incompatible
-	k8s.io/code-generator v0.30.6
-	sigs.k8s.io/controller-tools v0.13.0
-	sigs.k8s.io/testing_frameworks v0.1.2
-)
+toolchain go1.22.9
```

Comment:
We don't want toolchain directives. Just put 1.22.9 directly to go directive, and remove this line.

Reply:
We should also bump go to 1.22.9 or 1.23.x. I think generate is failing because of that.

Reply:
let's do this in separate PR, I was able to fix the issue without needing to do that.